### PR TITLE
Update to Dev CoC - Negligence criteria

### PR DESCRIPTION
**Pull request by suspect15**
Update to the Developer Code of Conduct Negligence criteria per DevCompliance following developer discussion.

### CODE REVIEW

File: docs/guidelines/developers/code-of-conduct.md [hunk 1]
```diff
@@ -58,7 +60,14 @@ Inactivity as a developer will be defined as:
...
+Maintenance of existing achievements is a developer's highest priority on RetroAchievements. Tickets shall be addressed as soon as possible. A developer shall be considered negligent and lose the developer role if they have either of the following:
+
+- 5 or more unaddressed tickets that are at least 1 month old.
+- Any unaddressed tickets that are at least 2 months old.
+
+An unaddressed ticket is defined as a ticket that has not had a comment by the developer in at least 2 weeks. If a developer cannot resolve a ticket with the information provided in the ticket and the reporter is unresponsive, the ticket may be closed as "Not enough information." If a developer is still working on a ticket, they must make a status update comment at least every 2 weeks for the ticket to be considered being actively addressed.
```
Comment (wescopeland) [inline docs/guidelines/developers/code-of-conduct.md]:
```suggestion
An unaddressed ticket is defined as a ticket that has not had a comment by the developer in at least 2 weeks. If a developer cannot resolve a ticket due to insufficient information in the ticket and the reporter is unresponsive, the ticket may be closed as "Not enough information." If a developer is still working on a ticket, they must provide a status update comment at least every 2 weeks for the ticket to be considered being actively addressed.
```

File: docs/guidelines/developers/code-of-conduct.md [hunk 2]
```diff
@@ -58,7 +60,14 @@ Inactivity as a developer will be defined as:
...
+
+The intent of the Negligence policy is to ensure developers are putting forth a good faith effort to prioritize and resolve their tickets. If there are extenuating circumstances that might prevent a developer from meeting the Negligence requirements, [DevCompliance](https://retroachievements.org/messages/create?to=DevCompliance) may grant an extension upon request.
```
Comment (wescopeland) [inline docs/guidelines/developers/code-of-conduct.md]:
```suggestion
The intent of the Negligence policy is to ensure developers make a good faith effort to prioritize and resolve their tickets. If there are extenuating circumstances that might prevent a developer from meeting the Negligence requirements, [DevCompliance](https://retroachievements.org/messages/create?to=DevCompliance) may grant an extension upon request.
```

File: docs/guidelines/developers/code-of-conduct.md [hunk 3]
```diff
@@ -58,7 +60,14 @@ Inactivity as a developer will be defined as:
 
 ### Negligence
 
-Alternatively, developer status will be removed if the developer has 10 or more tickets that are at least 2 months old. If the developer has any active claims during this period, they will be given an additional 30 days or until the earliest claim renewal (whichever comes first) to resolve all 10 tickets. Not doing so will result in the developer falling into inactive status and the claim(s) being released.
+Maintenance of existing achievements is a developer's highest priority on RetroAchievements. Tickets shall be addressed as soon as possible. A developer shall be considered negligent and lose the developer role if they have either of the following:
```
Comment (wescopeland) [inline docs/guidelines/developers/code-of-conduct.md]:
```suggestion
**A developer's highest priority on RetroAchievements is maintaining the achievements they've created.** Tickets shall be addressed as soon as possible. A developer shall be considered negligent and lose the developer role if they have either of the following:
```

**Review by wescopeland — COMMENTED** [review]
Looks good - minor suggestions for clarity.

**Comment by suspect15** [comment]
Great recommendations.  I committed both as suggested.

**Comment by wescopeland** [comment]
One suggestion that may have been missed and then I think we'll be good to go: https://github.com/RetroAchievements/docs/pull/103#discussion_r1740177192

**Comment by suspect15** [comment]
I did indeed miss that.  Updated.  Thanks for the back up.

**Review by wescopeland — APPROVED** [review]
Looks good 👍

File: docs/guidelines/developers/code-of-conduct.md [hunk 4]
```diff
@@ -7,6 +7,8 @@ description: Essential guidelines for RetroAchievements developers, including re
 
 This document covers developers' rights, responsibilities, and rules for fair play.
 
+[[toc]]
+
 **If you are not a dev, check [how to become a developer](/developer-docs/how-to-become-an-achievement-developer).**
 
 **See Also**: [Users Code of Conduct](/guidelines/users/code-of-conduct).
```
Comment (Sr78879516) [inline docs/guidelines/developers/code-of-conduct.md:14]: PPSSPP.org

Comment (Sr78879516) [reply docs/guidelines/developers/code-of-conduct.md:14]: Kaouppsspp.org

**Pull request merged**